### PR TITLE
15 4pt display list of items on the homepage student a

### LINKED ISSUE #15: [4pt]  - Display list of items on the Homepage - Student A
Create the main part of the homepage that keeps the layout from the wireframe: 

Screenshot 2021-06-27 at 21 50 23

When the page loads, the webapp retrieves data from  the selected API and shows the list of items on screen with the basic data (e.g. title + image).

Prepare also "Comments" and "Reservations" buttons. They should be doing nothing - just being displayed.

This task **does not** include displaying the number of likes for each item.

This task **does not** include the counter of the items.

### PR DESCRIPTION
**Pull request by SSEKPIUS**
The JavaScript capstone project is about building our own web application based on an external API. we will select an API that provides data about a topic that we like and then build the web app around it.

### CODE REVIEW

File: src/modules/Dom.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import ht2 from './images/heart2.png';
+// import countFoods from './countFoods.js';
+// import { addlikes, getlikes } from './likes.js';
+// import { showPopup } from './popup.js';
```
Comment (Ambient567) [inline src/modules/Dom.js:4]: I suggest you do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history

File: src/modules/Dom.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+const container = document.querySelector('.food-container');
+
+// eslint-disable-next-line no-unused-vars
```
Comment (Ambient567) [inline src/modules/Dom.js:8]: We set up linters to assist us in writing standardized code. Please enable linters and try to resolve the issue. If a rule proved difficult to solve, you are only permitted to disable that rule. Even so, you'll need to determine whether the rule you're disabling is something that can be easily solved or not.

**Review by Ambient567 — COMMENTED** [review]
Hi @SSEKPIUS 

Good job so far! 👍
There are some issues that you still need to work on to go to the next project, but you are almost there!

Highlights
No linter errors ✔️
Good commit messages ✔️

Required Changes ♻️
Check the comments under the review.

Cheers and Happy coding! 👏 👏 👏

File: src/modules/Dom.js (hunk 1 above)
Comment (SSEKPIUS) [reply src/modules/Dom.js:4]: This is resolved, thank you @Ambient567

File: src/modules/Dom.js (hunk 2 above)
Comment (SSEKPIUS) [reply src/modules/Dom.js:8]: This was an intermediary, thank you

**Pull request merged**